src/test/java/com/softserveinc/ita/rozetka_test/OrderTest.java

```diff
@@ -5,20 +5,56 @@
 import com.softserveinc.ita.rozetka.page_objects.HomePage;
 import com.softserveinc.ita.rozetka.page_objects.OrderPage;
 import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
 
 public class OrderTest extends TestRunner {
 
-    //This added for future tests
     private OrderPage orderPage;
     private BasketPage basketPage;
 
-    //This added for future tests
     @BeforeMethod
     public void addProductToBasketOpenOrder() {
         orderPage = new HomePage()
                 .openHomePage()
                 .openProductByNumber(1)
                 .addProductToBasket()
                 .openOrderPage();
+
+        assertEquals(orderPage.getHeaderText(), "Оформлення замовлення", "Page header should be" +
+                " 'Оформлення замовлення'");
+    }
+
+    @Test
+    public void testEditOrder() throws Exception {
+        basketPage = orderPage.startEditingProductsInBasket();
+
+        var orderProductCount = basketPage.getProductCount(1);
+        var orderProductSum = basketPage.getOrderProductSum(1);
+
+        basketPage.increaseProductCount(1);
+        var changedOrderProductCount = basketPage.getProductCount(1);
+        var changedOrderProductSum = basketPage.getOrderProductSum(1);
+
+        basketPage.openOrderPage();
+        assertEquals(changedOrderProductCount, orderProductCount * 2, "the new count for the product " +
```

Comment:
combine "the new..." and " should ..." into single string, move to new line, same for similar

Reply:
fixed